src/main/java/uk/gov/hmcts/darts/common/entity/HearingEntity.java

```diff
@@ -78,7 +78,7 @@ public class HearingEntity extends CreatedModifiedBaseEntity {
     @Transient
     private boolean isNew; //helper flag to indicate that the entity was just created, and so to notify DAR PC
 
-    @ManyToMany
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
I think there's something not right with the modelling of this ManyToMany relationship.
It looks strange to me because one side should be the owning parent side, so you would expect to have the JoinTable annotations like we have here, but if you navigate to the other side. It has similar, whereas I'd expect to see `mappedBy` attribute of ManyToMany being used by the other side of the bidirectional relationship.

Reply:
It does look a bit odd - do you think that could be covered under a separate tech debt ticket so as to not impede DMP-1090?

Reply:
I'd be worried about performance here. Looks like this would affect the Advanced Search etc, which is not necessarily interested in events?

Reply:
As dicussed let's handle this as a separate task.

Reply:
This change is reverted per https://github.com/hmcts/darts-api/pull/543.